### PR TITLE
update _integration_suite_test.go scaffold for underlying API changes

### PR DESCRIPTION
**Pull request by drnic**

### CODE REVIEW

**Comment by shanks3012** [comment]
Could you explain the changes to bindata.go a bit? @drnic

**Comment by drnic** [comment]
I ran the go generate command.

**Comment by drnic** [comment]
You can run it yourself and commit to master, then close this PR.

**Comment by kardolus** [comment]
@drnic can you elaborate a bit? Not sure what you mean by `go generate` in relation to `bindata.go`

**Comment by drnic** [comment]
https://github.com/cloudfoundry/libbuildpack/blob/master/packager/README.md

**Comment by drnic** [comment]
Also see https://github.com/cloudfoundry/libbuildpack/issues/33

File: packager/scaffold/src/LANGUAGE/integration/_integration_suite_test.go [hunk 1]
```diff
@@ -120,14 +120,15 @@ func AssertUsesProxyDuringStagingIfPresent(fixtureName string) {
...
-			traffic, built, err := cutlass.InternetTraffic(
+			traffic, built, logs, err := cutlass.InternetTraffic(
 				bpDir,
 				filepath.Join("fixtures", fixtureName),
 				bpFile,
 				[]string{"HTTP_PROXY=" + proxy.URL, "HTTPS_PROXY=" + proxy.URL},
 			)
 			Expect(err).To(BeNil())
 			Expect(built).To(BeTrue())
+			Expect(logs).To(BeEmpty())
```
Comment (ndon55555) [inline packager/scaffold/src/LANGUAGE/integration/_integration_suite_test.go]: Why are logs expected to be empty?

File: packager/scaffold/src/LANGUAGE/integration/_integration_suite_test.go [hunk 2]
```diff
@@ -160,5 +161,6 @@ func AssertNoInternetTraffic(fixtureName string) {
 		Expect(err).To(BeNil())
 		Expect(built).To(BeTrue())
 		Expect(traffic).To(BeEmpty())
+		Expect(logs).To(BeEmpty())
```
Comment (ndon55555) [inline packager/scaffold/src/LANGUAGE/integration/_integration_suite_test.go]: Why are logs expected to be empty?

**Comment by ryanmoran** [comment]
I've resolved these issues and a couple more in https://github.com/cloudfoundry/libbuildpack/commit/1647f144742d9c2888cc35af88b2ac8156069ad4. Please reopen this if the issue you are describing here persists.

**Pull request closed without merging**